src/main/scala/Configs.scala

```diff
@@ -231,9 +216,16 @@ class BaseConfig extends Config (
         true
       }
       case NExtInterrupts => 2
+      case ExtMMIOPorts => AddrMap()
+/*
+        AddrMap(
+          AddrMapEntry("cfg", MemRange(0x50000000L, 0x04000000L, MemAttr(AddrMapProt.RW))),
+          AddrMapEntry("ext", MemRange(0x60000000L, 0x20000000L, MemAttr(AddrMapProt.RWX))))
+*/
       case NExtMMIOAXIChannels => 0
       case NExtMMIOAHBChannels => 0
       case NExtMMIOTLChannels  => 0
+      case NExtBusAXIChannels => 0
```

Comment:
Shouldn't this be a derived parameter from ExtMMIOPorts? Also, if we have this, can't we get rid of NExtMMIOAXIChannels?